tools/distrib/python/xds_protos/setup.py

```diff
@@ -41,7 +38,7 @@
 SETUP_REQUIRES = INSTALL_REQUIRES + ['grpcio-tools']
 setuptools.setup(
     name='xds-protos',
-    version='0.0.5',
+    version='0.0.8',
```

Comment:
Why the jump from 5 to 8?

Reply:
To test Bazel installation, I tried 0.0.6 to remove grpcio as a dependency. It works, but feels a bit hacky.

0.0.7 added grpcio back into requirements. 0.0.8 added the `bdist_wheel` for Python 2.